zha/application/platforms/sensor/__init__.py

```diff
@@ -1947,3 +1949,14 @@ class DanfossMotorStepCounter(Sensor):
     _attribute_name = "motor_step_counter"
     _attr_translation_key: str = "motor_stepcount"
     _attr_entity_category = EntityCategory.DIAGNOSTIC
+
+
+@MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ELECTRICAL_CONDUCTIVITY)
+class ElectricalConductivity(Sensor):
+    """Electrical Conductivity sensor."""
+
+    _attribute_name = "measured_value"
+    _attr_device_class: SensorDeviceClass = SensorDeviceClass.CONDUCTIVITY
+    _attr_state_class: SensorStateClass = SensorStateClass.MEASUREMENT
+    _attr_translation_key: str = "electrical_conductivity"
+    _attr_native_unit_of_measurement = ELECTRICAL_CONDUCTIVITY_DECISIEMENS_PER_METER
```

Comment:
Let's move this above most of the manufacturer-specific entities (i.e. before `TimeLeft`, so above line `1610` ).

Since it's a measurement sensor that also uses `measured_value` as the attribute, it might make sense to place below `FormaldehydeConcentration`, so in line `1391`/`1392`.

Reply:
Done.